### DISCUSSION TITLE
[Docker] Error Generating Database / Add -vvv Flag for Logging

### DISCUSSION THREAD

**Discussion by SkyeRangerDelta** [shlinkio/shlink#1838] [Help wanted] [answered]
Good now,

I'm trying to get my Shlink instance off the ground using the Docker image (Docker Hub (anon) - `shlinkio/shlink:stable`) via Portainer and while I've managed to get everything squared away for the most part; I keep smacking into this dreaded error on start causing an instant container crash:
```
Initializing database if needed...
[ERROR] Error generating database.. Run this command with -vvv to see specific error info.
```
I've got my ENV variables configured as per the docs for a MySQL server running on a different VM:
```
DB_DRIVER | mysql
DB_HOST | 192.168.50.102
DB_NAME | Shlink_DB
DB_PASSWORD | ******************
DB_PORT | 9114
DB_USER | shlink
```
I cannot for the life of me figure out how to add the `-vvv` flag so the container log will spit out something useful. It's also possible that I'm just missing something and I need another pair of eyes.

Thank you in advance,
Skye

**Comment by acelaya** [comment, marked as answer]
https://shlink.io/documentation/troubleshooting/#troubleshoot-errors-when-booting-up-the-docker-image

**Comment by SkyeRangerDelta** [comment]
Absolutely stellar. Thank you - just need to get re-oriented.

**Reply by acelaya** [reply]
I think it shouldn't be hard to make Shlink suggest using that env var instead of using `-vvv`, when running in a context where the latter is not possible https://github.com/shlinkio/shlink-installer/issues/193

**Reply by SkyeRangerDelta** [reply]
Follow up question - Shlink seems to not like using an IP address for an external database engine. I run Nginx Proxy Manager for handling public connections via fqdn - but since it requires port specificity for a proxy host, it throws off Shlink's connection. When I do specify an IP, Shlink rewrites it to whatever localhost resolves to (x.108 instead of x.102) ...which is not the same VM.

Is there a way around this?

**Reply by acelaya** [reply]
Can I suggest you open a separated "help request" for this other topic? Otherwise, it will be harder for others to find solutions to similar problems.

That said, I'm not familiar with everything you mentioned there, but Shlink should have no problem connecting to a database via IP address. Anything related with network resolution is not handled by Shlink itself.
